Core/ClientSMLSWIG/Python/pyproject.toml

```diff
@@ -0,0 +1,110 @@
+[project]
+name = "soar-sml"
+description = "Raw SML Bindings to the Soar Cognitive Architecture"
+version = "9.6.2.dev2"
+license = "BSD"
+maintainers = [
+    { name = "Nathan Glenn", email = "garfieldnate@gmail.com" },
+    { name = "Center for Integrated Cognition" }
+]
+authors = [
+    { name = "John E. Laird" },
+    { name = "Allen Newell" },
+    { name = "Paul Rosenbloom" },
+
+    { name = "A. Mininger" },
+    { name = "A. Nickels" },
+    { name = "A. Turner" },
+    { name = "A. Nuxoll" },
+    { name = "A. Deschamps" },
+    { name = "R. Marinier" },
+    { name = "B. Stearns" },
+    { name = "C. Dettmering" },
+    { name = "D. Ray" },
+    { name = "D. Pearson" },
+    { name = "I. Hines" },
+    { name = "J. Crossman" },
+    { name = "J. Boggs" },
+    { name = "J. Voigt" },
+    { name = "J. Xu" },
+    { name = "J. Li" },
+    { name = "K. Coulter" },
+    { name = "L. Goeddel" },
+    { name = "M. Lanting" },
+    { name = "M. Assanie" },
+    { name = "M. Tinkerhess" },
+    { name = "M. Bloch" },
+    { name = "M. Schmidt" },
+    { name = "N. Derbinsky" },
+    { name = "N. Glenn" },
+    { name = "N. Gorski" },
+    { name = "N. Timpko" },
+    { name = "S. Wintermute" },
+    { name = "S. Lathrop" },
+    { name = "S. Wallace" },
+    { name = "S. Nason" },
+    { name = "S. Mohan" },
+    { name = "S. Jones" },
+    { name = "Y. Wang" },
+    { name = "J. Kirk" },
```

Comment:
I feel bad now that I stuck jkirk and rwray at the bottom by accident 😅 Could you put them in their alphabetical location?